apps/system/js/quick_settings.js

```diff
@@ -336,6 +312,7 @@
                 window.dispatchEvent(
                   new CustomEvent('request-enable-bluetooth'));
               }
+              this.bluetooth.dataset.initializing = 'true';
```

Comment:
Once we make the change, the line can be removed because the state will be updated based on the current adapter state.